modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java

```diff
@@ -324,4 +341,16 @@ private void validateTerraformScript(Ocl ocl) {
         }
     }
 
+    private String generateJsonObjectSchemaString(List<DeployVariable> variables) {
```

Comment:
Everywhere we need the JsonObjectSchema directly instead of string.

Reply:
The above comments have been fixed, pls check.